### PR TITLE
Fix part of #2205: Clean up alternative versions of profile_chooser_profile_view to match each other.

### PR DESCRIPTION
**Pull request by Russ741**
## Explanation

This PR:
- resolves unintentional differences between versions of profile_chooser_profile_view
- pushes intentional differences from the layout files into lower-level resource files (i.e. dimens).

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [ ] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [ ] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by Russ741** [comment]
@BenHenning I've created the draft PR with the first few commits. The revision history looks a bit strange right now - did I make a mistake with git?

File: app/src/main/res/values-sw600dp/dimens.xml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <dimen name="profile_chooser_profile_view_circular_image_size">108dp</dimen>
+</resources>
```
Comment (BenHenning) [inline app/src/main/res/values-sw600dp/dimens.xml]: Nit: please add missing EOF newline.

File: app/src/main/res/layout-land/profile_chooser_profile_view.xml [hunk 2]
```diff
@@ -39,8 +39,8 @@
 
       <de.hdodenhof.circleimageview.CircleImageView
         android:id="@+id/profile_avatar_img"
-        android:layout_width="72dp"
-        android:layout_height="72dp"
+        android:layout_width="@dimen/profile_chooser_profile_view_circular_image_size"
```
Comment (BenHenning) [inline app/src/main/res/layout-land/profile_chooser_profile_view.xml]: It seems this is always corresponding to the profile avatar image. Perhaps: ``profile_chooser_avatar_circular_image_size`` might be clearer?

**Review by BenHenning — CHANGES_REQUESTED** [review]
Thanks @Russ741! This generally LGTM, so I think you can go ahead and move this to full review if you don't plan on making any other changes.

The last git commit looks like you may have made changes directly to your fork's copy of the profile-chooser-oobe-fixes branch and then pulled those in locally (which would trigger a merge). Not exactly sure what would have caused that, though.

File: app/src/main/res/layout-land/profile_chooser_profile_view.xml (hunk 2 above)
Comment (Russ741) [reply app/src/main/res/layout-land/profile_chooser_profile_view.xml]: Done.

**Comment by rt4914** [comment]
@Russ741 You will need to mark this PR as "Ready for Review", currently its in `DRAFT` mode.

**Comment by Russ741** [comment]
I'm closing this pull request because my approach to flattening the alternative resources for the profile_chooser_profile_view layout was infeasible. Specifically, I couldn't find a good way of combining:
1) behavior that selects between two values based on data binding (e.g. hasProfileEverBeenAddedValue)
2) extracting attributes that vary between alternative layouts (e.g. the CircleImageView layout_height) to resources (e.g. profile_chooser_avatar_circular_image_size)
3) Android named constants (e.g. Gravity.CENTER_HORIZONTAL)

The only solution I found was to assign a raw integer (with a comment indicating that it corresponds to Gravity.CENTER_HORIZONTAL) to the resource representing the layout's gravity, which is not good for code quality or readability.

**Pull request closed without merging**